lib/charms/layer/apache_zeppelin.py

```diff
@@ -231,15 +248,12 @@ def wait_for_stop(self, timeout):
         start = datetime.now()
         while utils.jps("zeppelin"):
             time.sleep(1)
-            if datetime.now() - start > timeout:
+            if (datetime.now() - start).seconds > timeout:
```

Comment:
Using time.time() to set start and this value would be cleaner, but those code is functional, if a bit odd.